packages/date-time/src/components/DatePicker/__snapshots__/DatePicker.test.tsx.snap

```diff
@@ -127,24 +127,6 @@ exports[`DatePicker renders default DatePicker correctly 1`] = `
                 &::-ms-clear {
                   display: none;
                 }
-                &::placeholder {
```

Comment:
ehhhh... are these styles moved somewhere else?

Reply:
Yes, it's moved to `@uifabric/styling` to be usable across all components that use a `placeholder`. 

[Link](https://github.com/OfficeDev/office-ui-fabric-react/blob/86c0805d690e568cfebdeb3c0dc6ec821749c199/packages/styling/src/styles/getPlaceholderStyles.ts) to file containing the change

Reply:
yes, but shouldn't the resulting snapshot be that the styles still show up?

Reply:
@micahgodbolt I shall get back on this. I am not sure how these styles got removed.

Reply:
The snapshots were not updated, the same reason why the CI build was failing. The updated snapshots show styles and pass build.